urllib3/request.py

```diff
@@ -1,13 +1,47 @@
 from __future__ import absolute_import
-try:
-    from urllib.parse import urlencode
-except ImportError:
-    from urllib import urlencode
 
 from .filepost import encode_multipart_formdata
 
+from .packages import six
 
-__all__ = ['RequestMethods']
+_Request = six.moves.urllib.request.Request
+urlencode = six.moves.urllib.parse.urlencode
+
+__all__ = ['RequestMethods', 'Request']
+
+
+class Request(_Request):
+    """
+    Currently used as a shim to allow us to work with the stdlib cookie
+    handling, which expects a :class:`urllib.request.Request`-like object.
+    """
+    def __init__(self, *args, **kwargs):
+        self._method = kwargs.pop('method', None)
```

Comment:
Force of habit, I guess. Given that this part of urllib3 isn't a public API, we can certainly not save it, and add it back later if we have a need.